test/asm/raw-macro-args.asm

```diff
@@ -22,7 +22,7 @@ STR EQUS "str\"ing"
 	printargs "literal \"\\\"", \ ; comment 2
 """multi-"line"
   ""string"" arg"""
-	printargs MUL(2.0\, 3.0)
+	printargs MUL(2.0, 3.0)
```

Comment:
Shouldn't both unescaped and escaped work?

Reply:
Yes, I just removed this comma because it was no longer necessary.

Reply:
Yes, but then I think both should be tested, for backwards compatibility's sake.

Reply:
I've added one in `a(b,c\,d)`.